### ISSUE TITLE
Helpers are not discovered

### ISSUE THREAD

**Issue by raido** [ember-learn/ember-cli-addon-docs#136]
I am struggling to get helpers section to display. 

```
{ file: 'addon/helpers/my-helper',
  line: 3,
  hide: '',
  class: 'addon/services/file-queue~FileQueue',
  module: undefined }
```

I just added few console.logs to lib/models/function.js on yuidoc package. It seems it matches helper to random service. That explains why I saw part of docblock from helper file in my service methods list. Helper’s docblock had @method defined.

**Comment by allthesignals** [comment]
I'm having issues with helpers too, maybe it's related.

I'm seeing:
> Build Error (DocsCompiler)
>
> Cannot read property 'filter' of undefined

In my case, if I comment the lines [here](https://github.com/ember-learn/ember-cli-addon-docs/blob/master/lib/broccoli/docs-compiler.js#L191) and [here](https://github.com/ember-learn/ember-cli-addon-docs/blob/master/lib/broccoli/docs-compiler.js#L207), the docs are generated no problem.

**Comment by allthesignals** [comment]
In the modules collection, the `helpers` keys is missing for me: 

```
{ id: 'addon/components/mapbox-gl',
    file: '/components/mapbox-gl',
    functions: [],
    variables: [],
    classes: [],
    components: [ [Object] ] }
```

**Comment by makepanic** [comment]
I'm seeing a similar error with an addon that has no helpers:

If I add a log to https://github.com/ember-learn/ember-cli-addon-docs/blob/master/lib/broccoli/docs-compiler.js#L60 it prints an object without the helpers property. Meaning `module.helpers.length` is called where `module.helpers` is undefined.

**Comment by pzuraq** [comment]
We've just upgraded and published changes to the API doc plugins that removed the notion of `helpers` altogether. Helpers can be classes or functions, so it makes more sense to just have a special nav item and have it point to whatever exists there (class or module).

Unfortunately we weren't able to merge the corresponding updates/fixes in the main ember-cli-addon-docs library due to failures on beta and canary, but we just did and just published the fix in a new version. Let us know if you're still having issues!
